requirements/requirements.txt

```diff
@@ -1,2 +1,4 @@
 Django>=4.1,<4.2
 wagtail>=4.1,<4.2
+django-storages
+psycopg2-binary
```

Comment:
Did you mean to remove django-bootstrap? I will merge as is, but please check and update if required.

Reply:
I need django-bootstrap too. I don't know why I forgot this. `django-storages` is needed for Azure deploy.